demos/demo4-photo-gallery/Demo.jsx

```diff
@@ -14,6 +14,18 @@ const Demo = React.createClass({
     this.setState({currPhoto: value});
   },
 
+  clickHandler(e){
+    var btnType = e.target.id === 'nbtn' ? 1 : 0;
```

Comment:
React typically doesn't use IDs for these. Can you create a handler + use this.click.bind(null, ...) or two handlers please?